README.md

```diff
@@ -27,6 +27,46 @@ script/bootstrap
 
 ## Running the application
 
+To run the application there are two options.
+
+For either option the frontend can further decide if it's requests should be
+sent to the locally running API or to a wiremock version. To do this we change
+the environment variable `APPROVED_PREMISES_API_URL` within .env to either of
+the following values:
+
+* Fake API (via wiremock) - <http://localhost:9092>
+* [Real API](https://github.com/ministryofjustice/hmpps-approved-premises-api) -
+  <http://localhost:8080>
+
+### 1. Using AP Tools
+
+In order to spin up a full stack of a working API and other [dependant
+services](./docker-compose.yml) we recommend using the [AP
+Tools](https://github.com/ministryofjustice/hmpps-approved-premises-tools).
+
+NB. The approach AP Tools takes solves a critical limitation for working in
+development. Due to how the frontend and API authenticate requests they both
+require access to _the same_ instance of hmpps-auth. This project is the focus
+of our development tooling across all CAS services and is most likely to receive
+future updates.
+
+After following the set up the common commands are:
+
+```bash
+ap-tools server start --local-ui --local-api
+```
+
+```bash
+ap-tools server start --local-ui --local-api
+```
+
+The service should then be available at <http://localhost:3000>
+
+[Log in credentials are documented within AP
+tools](https://github.com/ministryofjustice/hmpps-approved-premises-tools#start-server).
+
+### 2. Manually
+
```

Comment:
Worth a small note about why we might do this? - Quicker initial startup time, and able to test features that aren't yet present in the API?